katas/src/lib.rs

```diff
@@ -13,6 +13,8 @@ use qsc::{
     PackageType, SourceContents, SourceMap, SourceName,
 };
 
+use qsc_data_structures::language_features::LanguageFeatures;
```

Comment:
Same here and through out the other uses. That should also remove the need to add `qsc_data_structures` to the Cargo.toml files.